src/infra/workload/releaseunit/modules/stamp/keyvault-secrets.tf

```
@@ -10,12 +10,14 @@ locals {
     "StorageAccount-Healthservice-ContainerName" = azurerm_storage_container.deployment_healthservice.name
     "StorageAccount-Healthservice-BlobName"      = local.health_blob_name
     "Global-StorageAccount-ConnectionString"     = data.azurerm_storage_account.global.primary_connection_string
-    "APPLICATIONINSIGHTS-CONNECTION-STRING"      = data.azurerm_application_insights.stamp.connection_string
-    "APPLICATIONINSIGHTS-ADAPTIVE-SAMPLING"      = var.ai_adaptive_sampling,
+    "ApplicationInsights-Connection-String"      = data.azurerm_application_insights.stamp.connection_string
+    "ApplicationInsights-Adaptive-Sampling"      = var.ai_adaptive_sampling,
     "CosmosDb-Endpoint"                          = data.azurerm_cosmosdb_account.global.endpoint
     "CosmosDb-ApiKey"                            = data.azurerm_cosmosdb_account.global.primary_key
     "CosmosDb-DatabaseName"                      = var.cosmosdb_database_name
-    "API-Key"                                    = var.api_key
+    "Api-Key"                                    = var.api_key
+    "LogAnalytics-WorkspaceId"                   = data.azurerm_log_analytics_workspace.stamp.workspace_id
+    "Managed-Identity-Clientid"                  = azurerm_kubernetes_cluster.stamp.kubelet_identity.0.client_id
```

```suggestion
    "Managed-Identity-ClientId"                  = azurerm_kubernetes_cluster.stamp.kubelet_identity.0.client_id
```

can we use **I**d instead of **id** please? :D